src/main/java/seedu/address/model/CategoryList.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import seedu.address.model.expense.Category;
+import seedu.address.model.expense.Expense;
+
+
+//@@author Jiang Chen
+/**
+ * Manage all Category class created during addition of expense.
+ * Duplicate is not allowed.
+ * */
+public class CategoryList {
+    public final HashMap<Category, ArrayList<Expense>> categoryList;
+
+    /**
+     * Create a HashMap of {@code Category}. The key is Category itself and the value is {@code expenseList}
+     * */
+    public CategoryList() {
+        this.categoryList = new HashMap<>();
+    }
+
+    /**
+     * Add a new {@code Category} into CategoryList.
+     * */
+    public void addCategory(Category category) {
+        ArrayList<Expense> newExpenseList = new ArrayList<>();
+        categoryList.put(category, newExpenseList);
+    }
+
+    /**
+     * This is a tentative method. It is used for testing categorization.
+     * */
+    public void addExpense(Category category, Expense expense) {
```

Comment:
Is this used in the actual implementation? If it is not, please comment it out

Reply:
Yes, I think it will be used in actual implementation. As it may not be the final implementation, I comment it as tentative. In this stage, it is mainly used for JUnit test. If it works well, I will use it in Add command. Sorry for confusion, I will change the comment.

Reply:
Please add a `TODO` so that we do not forget about this code

Reply:
Actually I think the addExpense() in addressbook is more likely a tentative one. I will add TODO there.